### PR TITLE
Add a reminding comment to keep etcd name in sync in etcd.manifest an…

### PR DESCRIPTION
**Pull request by wenjiaswe**
…d migrate-if-needed.sh

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind documentation

**What this PR does / why we need it**:
Add a comment in configure-helper.sh [1] and migrate-if-needed.sh [2] to keep the change of etcd name in sync.

[1]: https://github.com/kubernetes/kubernetes/blob/0891f69f5ec27cdbcb6336ce3cfe6efdcc1d1e05/cluster/gce/gci/configure-helper.sh
[2]: https://github.com/kubernetes/kubernetes/blob/0891f69f5ec27cdbcb6336ce3cfe6efdcc1d1e05/cluster/images/etcd/migrate-if-needed.sh

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
N/A
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1594,6 +1594,7 @@ function start-kube-proxy {
 # $4: value for variable 'cpulimit'
 # $5: pod name, which should be either etcd or etcd-events
 function prepare-etcd-manifest {
+  # NOTE: Keep host_name here in sync with ETCD_NAME in https://github.com/kubernetes/kubernetes/blob/2e7996e3e2712684bc73f0dec0200d64eec7fe40/cluster/images/etcd/migrate-if-needed.sh#L46
```
Comment (wojtek-t) [inline cluster/gce/gci/configure-helper.sh:1597]: Hmm - the vendor/ changes seem unrelated - please revert.

Comment (wenjiaswe) [reply cluster/gce/gci/configure-helper.sh:1597]: Oops, sorry, that was a mistake. I closed this one and have this new one instead: #91964.
